x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/advanced_view/detector_list.tsx

```diff
@@ -166,3 +172,7 @@ const DuplicateDetectorsWarning: FC<{ validation: Validation }> = ({ validation
     </Fragment>
   );
 };
+
+const StringifiedDetector: FC<{ detector: Detector }> = ({ detector }) => {
+  return <div data-test-subj="detectorIdentifier">{detectorToString(detector)}</div>;
```

Comment:
Should this subject be `mlDetectorIdentifier`?

Reply:
See above, renamed in dfe2e17